Woof/Woof/View/ReusableViews/TextEditorWithPlaceholder.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+/// A view that displays a text input field with a prompt text message for a user.
+struct TextEditorWithPlaceholder: View {
+    /// The text to be shown in the text input field.
+    @Binding var message: String
+
+    /// The text that is shown in the text input field when it’s empty.
+    let placeholder: String
+
+    var body: some View {
+        ZStack(alignment: .topLeading) {
+            if message.isEmpty {
+                Text(placeholder)
+                    .padding(AppStyle.UIElementConstant.spacingBetweenElements)
+                    .foregroundColor(Color.gray)
+            }
+            TextEditor(text: $message)
+                .opacity(message.isEmpty ? opacityValueForEmptyTextEditor : opacityValueForFilledTextEditor)
+        }
+    }
+}
+
+// MARK: - Private interface
+
+private let opacityValueForEmptyTextEditor = 0.25
```

Comment:
Can we make it simpler?

Smth like:
```suggestion
private let emptyTextEditorOpacity = 0.25
```
?